backend/server.js

```diff
@@ -961,10 +955,37 @@ app.get("/qInstances/:qiid/:filename", function(req, res) {
                 if (!_(info).has("clientFiles")) {
                     return sendError(res, 500, "Question does not have clientFiles, qid: " + fileInfo.qid);
                 }
```

Comment:
We should move this lower so that you can have templated client files without non-templated files.